webextensions/api/tabGroups.json

```diff
@@ -0,0 +1,697 @@
+              }
+            }
+          },
+          "windowId": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "89"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "139"
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          }
+        },
+        "get": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "89"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "139"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "groupId": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "89"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "139"
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          }
+        },
+        "move": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "89"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "139"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "groupId": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "89"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "139"
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
```

Comment:
`webextensions.api.tabGroups.move.groupId` does not need to be documented separately, as it is a core part of the API definition.

```suggestion
```